pkgs/tools/typesetting/tex/texlive/bin.nix

```diff
@@ -309,6 +309,10 @@ xindy = stdenv.mkDerivation {
   name = "texlive-xindy.bin-${version}";
 
   inherit (common) src;
+
+  # If unset, xindy will try to mkdir /homeless-shelter
+  HOME = "/tmp/homeless-shelter";
```

Comment:
I'm not sure if this is sandbox-friendly. Perhaps it would be better to patch the build scripts to not do the funny business.

Reply:
Alternatively (although avoiding it might be better) can probably do something like setting `HOME=$(mktemp -d)` in preConfigure or something.

Reply:
I always do `HOME = .` in such cases.